pandas/tests/frame/test_join.py

```diff
@@ -31,6 +31,11 @@ def right():
     return DataFrame({'b': [300, 100, 200]}, index=[3, 1, 2])
 
 
+@pytest.fixture
+def right_non_unique():
```

Comment:
Why'd you make this a fixture? If it's only used in one place then I'd prefer it be defined where it's used.

Reply:
Well, the below test is just the bare minimum, I think `test_join` needs to cover many more situations. But I will pull it into the method for now.